### DISCUSSION TITLE
What is the version of VTM in results?

### DISCUSSION THREAD

**Discussion by cucNaifuXue** [InterDigitalInc/CompressAI#198] [Q&A]
Hi,
Thanks for the experiment results, which helped me a lot!

Could you release the version of VTM in your results [results/image/kodak/vtm.json](https://github.com/InterDigitalInc/CompressAI/blob/master/results/image/kodak/vtm.json) ?

I found cods in [examples/run-benchmarks.sh](https://github.com/InterDigitalInc/CompressAI/blob/2af5220f84657199e57a72b88fa7ea7e73016e67/examples/run-benchmarks.sh)
```
# VTM
# edit below to provide the path to the chosen version of VTM
# _VTM_SRC_DIR="${HOME}/vvc/vtm-9.1"
```
Is the version 9.1?

Thank you for your reply!

**Comment by fracape** [comment]
Hi, yes but that's a good point, I should have added the version in the result files, like I did for video. I'll try to update everything soon if I have time (with more up-to-date versions of HM/VTM and AV1)
